iacrcc/iacrdoc.tex

```diff
@@ -148,18 +142,24 @@ \subsection{Title}
 {\tt running} & The running title displayed in the headers.\\
 {\tt onclick} & Define what to do when clicking on the title. This could be used to point to a webpage associated with this paper. \\
 {\tt subtitle} & Provide a subtitle.\\
+{\tt plaintext} & A text version of the title.\\
 \end{tabular}
 \end{center}
-\noindent  Do not use author-defined macros in arguments to
-\texttt{\textbackslash title}. An example using all the optional
-options would look like:
+\noindent An example using all the optional options would look like:
 
 \begin{verbatim}
 \title[running  = {IACR Communications in Cryptology Class},
        onclick  = {https://github.com/IACR/latex},
-       subtitle = {A Template}
-      ]{How to Use the IACR Communications in Cryptology Class}
+       subtitle = {A Template},
+       plaintext = {How to Use this class}
+      ]{How to Use this class\thanks{Thanks Mom!}}
```

Comment:
I think this example is unfortunate: authors should *not* use the \thanks macro. We provide the footnote option in addauthor or the genericfootnote macro. Maybe we should even disallow the \thanks macro?

Reply:
I'm not completely clear on why we need to disable \thanks so I am creating a separate issue about this (I think it's important to resolve).